include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -263,6 +263,9 @@ SECTIONS
 
     GROUP_START(RAMABLE_REGION)
 
+#if defined(CONFIG_RAM_FUNCTION)
```

Comment:
I think this is put _too_ early.
Would it be more correct to add it right before the CONFIG_APP_SHARED_MEM section?

Reply:
It does look like it's too early. Right below we have something saying

"Must be first in ramable region"